source/Item.cpp

```diff
@@ -2,86 +2,86 @@
 
 namespace dungeon {
 
-Item::Item() {
-	hpAdd = atkAdd = 0;
-	hpPercentAdd = atkPercentAdd = 0;
-	critRateAdd = critDamageAdd = 0;
-}
+WreckingBar::WreckingBar() {
+	type = ClassType::ANY;
+	name = "Wrecking Bar";
 
-Item& Item::operator+=(const Item& item) {
-	hpAdd += item.hpAdd;
-	atkAdd += item.atkAdd;
-	hpPercentAdd += item.hpPercentAdd;
-	atkPercentAdd += item.atkPercentAdd;
-	critRateAdd += item.critRateAdd;
-	critDamageAdd += item.critDamageAdd;
-	return *this;
+	attr.baseAtk = 100;
+	attr.critRatePercent = 5;
```

Comment:
Still, better to add a constructor in the base class instead of so many duplicated codes.